tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js

```diff
@@ -10,31 +10,61 @@ const ContextualButton = ContextualPopupDecorator(Button);
 // runs the same way
 spotlight.setPointerMode(false);
 
-const renderPopup = () => (
-	<Button id="popupButton">Hello Contextual Popup</Button>
+const renderPopup1 = () => (
+	<Button id="popupButton1">Hello Contextual Popup1</Button>
+);
+
+const renderPopup2 = () => (
+	<Button id="popupButton2">Hello Contextual Popup2</Button>
+);
+
+const renderPopup3 = () => (
+	<Button id="popupButton3">Hello Contextual Popup3</Button>
 );
 
 class app extends Component {
 
 	state = {
-		open: false
+		button1Open: false,
+		button2Open: false,
+		button3Open: false
 	};
 
 	clickHandler = (st) =>  this.setState(st);
 
 	render () {
-		const {open} = this.state;
+		const {button1Open, button2Open, button3Open} = this.state;
+
 		return (
 			<div {...this.props}>
 				<ContextualButton
 					id="button1"
-					onClick={() => this.clickHandler({open: !open})}
-					onClose={() => this.clickHandler({open: false})}
-					open={open}
-					popupComponent={renderPopup}
+					onClick={() => this.clickHandler({button1Open: !button1Open})}
+					onClose={() => this.clickHandler({button1Open: false})}
+					open={button1Open}
+					popupComponent={renderPopup1}
+				>
+					Contextual Popup Button1
+				</ContextualButton>
+				<ContextualButton
+					id="button2"
+					onClick={() => this.clickHandler({button2Open: !button2Open})}
+					onClose={() => this.clickHandler({button2Open: false})}
+					open={button2Open}
+					popupComponent={renderPopup2}
 					spotlightRestrict="self-only"
```

Comment:
I see that, in the view,  the difference between ContextualPopupDecorator1 and ContextualPopupDecorator2 is `spotlightRestrict="self-only"` , but I don't see any tests related to spotlightRestrict.  Shouldn't be there tests related to this?
If yes, please add them.
If not, I don't see the point of having both ContextualPopupDecorator1 and ContextualPopupDecorator2 in the view as both have the same behavior for the existing added tests.